sal_interfaces/MTAlignment/MTAlignment_Commands.xml

```diff
@@ -21,6 +21,39 @@
         <Count>1</Count>
     </item>
 </SALCommand>
+<SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <Version>3.7.1</Version>
+    <Author>Colin Winslow</Author>
+    <EFDB_Topic>MTAlignment_command_correctionloop</EFDB_Topic>
+    <Alias>measurePoint</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://sal.lsst.org</Explanation>
+    <item>
+        <EFDB_Name>target</EFDB_Name>
+        <Description>name of target, M2 or CAM</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>dimensionless</Units>
```

Comment:
I think the correct value for Units in this case is `unitless` (https://ts-xml.lsst.io/sal_constraints_and_recommendations.html?highlight=units#sal-topic-and-parameter-description-and-unit-fields)